src/actions/articleCategoryAction.js

```diff
@@ -0,0 +1,34 @@
+import { ACTIONS } from './actionTypes';
+import axios from 'axios';
+
+/**
+ * @description - articles categories response function
+ * @param {string} actionType - type of action
+ * @param {object} category - article category
+ * @returns {object}
+ */
+export const categoryActionResponse = (actionType, category) => ({
+  type: actionType,
+  payload: category
+});
+
+/**
+ * @description Request to the API to get only 6 articles categories
+ * @returns {object} dispatch object
+ */
+export const getArticlesCategory = () => async(dispatch) => {
+  try {
+    categoryActionResponse(ACTIONS.LOADING, 'started');
+    const response = await axios.get(`${process.env.API_BASE_URL}/categories?size=6`);
+    const { data: { data } } = response;
+    if (response.status === 404) {
+      dispatch(categoryActionResponse(ACTIONS.GET_ARTICLES_CATEGORY_FAILURE, 'Category not found'));
+      categoryActionResponse(ACTIONS.STOP_LOADING, 'stopped');
+    }
+    dispatch(categoryActionResponse(ACTIONS.GET_ARTICLES_CATEGORY_SUCCESS, data));
+    categoryActionResponse(ACTIONS.STOP_LOADING, 'stopped');
+  } catch (err) {
+    categoryActionResponse(ACTIONS.STOP_LOADING, 'stopped');
+    return err;
```

Comment:
#### Nitpick
- Please lets not return errors directly, let's use the dispatched action.

Reply:
okay thanks